lib/random.d.ts

```diff
@@ -37,25 +37,22 @@ declare module "@offirmo/random" {
 		static die(sideCount: number): (engine: Engine) => number;
 		static dice(sideCount: number, dieCount: number): (engine: Engine) => number;
 		static uuid4(engine: Engine): string;
-		static string(engine: Engine, length: number): string;
-		static string(pool: string, length: number): (engine: Engine, length: number) => string;
+		static string(pool?: string): (engine: Engine, length: number) => string;
 		static hex(upperCase?: boolean): (engine: Engine, length: number) => string;
 		static date(start: Date, end: Date): (engine: Engine) => Date;
 
 		integer(min: number, max: number): number;
 		real(min: number, max: number, inclusive: boolean): number;
-		bool(percentage?: number): (engine: Engine) => boolean;
+		bool(percentage?: number): boolean;
 		bool(numerator: number, denominator: number): boolean;
-		pick<T>(engine: Engine, array: Array<T>, begin?: number, end?: number): T;
-		picker<T>(array: Array<T>, begin?: number, end?: number): (engine: Engine) => T;
```

Comment:
picker doesn't exist in the alternative syntax. It looks to correct a mistake - `pick` takes engine as a parameter in the initial function list instead of return a function from engine to T.